src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs

```diff
@@ -41,7 +41,7 @@ public async Task<IActionResult> Get(Guid calendarEventId, [FromHeader(Name = Co
 
     [HttpGet]
     [ProducesResponseType(typeof(GetCalendarEventsQueryResult), StatusCodes.Status200OK)]
-    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
We should return empty results when no data matches the filters.

Reply:
ok